### DISCUSSION TITLE
API

### DISCUSSION THREAD

**Discussion by MiningCryptoLive** [dreams-money/merged-mining-pool#15] [Q&A]
Hello Again,

Trying to create a web ui for the merged-mining-pool. How does the api work? Or can you point me in a direction to learn it?

Thanks

**Comment by dreams-money** [comment]
The API is pretty simple - only 3 routes.  It's lacking docs at the moment, but the code can be read.

Here's a good starting point - https://github.com/dreams-money/merged-mining-pool/blob/master/api/server.go#L62
